assets/js/localization.js

```diff
@@ -207,18 +220,42 @@ function getLocales() {
         var localePairs = [];
         for(var code in locales) {
             localePairs.push([code, locales[code]]);
+            pageLocales[code] = locales[code];
         }
         localePairs = localePairs.sort(function (a, b) {
             return a[1].toLowerCase().localeCompare(b[1].toLowerCase());
         });
         $('.localeSelect').empty();
+        $('#localeDropdown').empty();
         $.each(localePairs, function () {
             $('.localeSelect').append(
                 $('<option></option>')
                     .val(this[0])
                     .text(this[1])
                     .prop('dir', rtlLanguages.indexOf(this[0]) !== -1 ? 'rtl' : 'ltr')
             );
+
+            $('#localeDropdown').append(
+                $('<li></li>').append(
+                    $('<a>', {
+                        rel: this[0] // eslint-disable-line id-length
+                    })
+                        .text(this[1])
+                        .prop('dir', rtlLanguages.indexOf(this[0]) !== -1 ? 'rtl' : 'ltr')
```

Comment:
I wonder if it would be worth not exporting `rtlLanguages` and instead a function like `isRtlLanguage` or something like that... I don't recall if we use `rtlLanguages` for any other purpose though.